webpack/index.d.ts

```diff
@@ -533,6 +595,10 @@ declare namespace webpack {
         new (paths: RegExp[]): Plugin;
     }
 
+    interface NamedModulesPluginStatic {
+        new (): Plugin;
+    }
```

Comment:
Possibly because `new NamedModulesPlugin()` will return `NamedModulesPlugin` instead of `Plugin`? What were the symptoms?